library/react/util/stringToId.js

```diff
@@ -1,11 +1,11 @@
 /**
  * Convert an input string such as a title to a valid html id
- * @param {string} inputString The string to conver to an html id string
+ * @param {string} inputString The string to convert to a html id string
```

Comment:
an before a 'h' or acronym is correct grammar?

Reply:
I think so too, but intellij said it was wrong with this message, "Use a instead of 'an' if the following word doesn't start with a vowel sound, e.g. 'a sentence', 'a university'.